presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -103,9 +112,10 @@
 public final class HttpRemoteTask
         implements RemoteTask
 {
-    private static final Logger log = Logger.get(HttpRemoteTask.class);
+    private static final Logger LOG = Logger.get(HttpRemoteTask.class);
```

Comment:
There are ~160 occurrences of `Logger log` vs ~20 occurrences of `Logger LOG`. I would stick with `log`, as upper case is rather used for constants. `Logger` object is not a data object, so it is hard to call it a constant.

Reply:
cc @nezihyigitbasi